### ISSUE TITLE
SDXL Turbo support

### ISSUE THREAD

**Issue by CHJ85** [easydiffusion/easydiffusion#1689]
Hi there. So now that SDXL Turbo is a thing and it can basically generate images on-the-fly as you type in the prompt, it'd be nice if we could enables this feature within Easy Diffusion.
So here's my solution. A "Turbo mode" check/uncheck button either next to the Generate button or the Models switcher drop-down menu. When checked, the generation process happens as you're typing the prompt.
In case you don't know what this new turbo model is all about, here's a demo: https://clipdrop.co/stable-diffusion-turbo
And here's the model download link: https://huggingface.co/stabilityai/sdxl-turbo/tree/main

**Comment by glor** [comment]
I agree that having an easy option for this could be beneficial. But in case someone needs the information I'll leave it here: You can already use turbo if you set Guidance Scale to 1, steps to 4, the sampler to Euler Ancestral and disable all the special features (at least that what I have tested so far).

**Comment by CHJ85** [comment]
@glor Really? See, I didn't even think that was a possibility.
I thought you still had to press the Generate button.
Because what really makes it "turbo" is that your image changes on-the-fly as you type in your prompt.

**Comment by Alexancj919** [comment]
The SDXL Turbo allows processing through much fewer steps, around 1-4, instead of dozens.  But it operates essentially the same.  It's "on the fly" in the sense that it is quick.  If you wrote a script that hit "Generate" after every key press, it would approximate how the clipdrop page works.  However, Stability.AI has access to processors that home users don't likely have - and hundreds/thousands of them.  They can cut down this response per keystroke from around one second to a fraction.

**Comment by AvidGameFan** [comment]
I've been running a couple of turbo models, and it's not fast enough (at least on my machine) to do it realtime, but it does cut the time in half.

For those with higher-end cards, this might be a possibility, generating an image every few seconds.  So, it'd be a slight pause as you type the prompt.  It probably doesn't make sense to regenerate for every keypress anyway.  But I think it'd be a larger change to get it to continuously update in-place like that (not sure if it could be done in a plugin, but maybe).
